### DISCUSSION TITLE
Welcome to CIS Discussions!

### DISCUSSION THREAD

**Discussion by arkodyuti** [GitHub-Campus-Program-India/CIS#135] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with participants during Code Innovation Series events. We hope that you:
  * Ask questions you’re wondering about.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Have questions while building your hackathon project? Start a discussion.

**Comment by Dhruv-240** [comment]
When would the winner be announced
